internal/provider/resource_ko_image_test.go

```diff
@@ -60,4 +60,21 @@ func TestAccResourceKoImage(t *testing.T) {
 			),
 		}},
 	})
+
+	// Test that working_dir can be set.
+	resource.Test(t, resource.TestCase{
+		ProviderFactories: providerFactories,
+		Steps: []resource.TestStep{{
+			Config: `
+			resource "ko_image" "foo" {
+			  importpath = "."
+			  working_dir = "./cmd/test"
```

Comment:
Thanks, that helps fix the test!

I'm not sure it helps with the reported module hell though, since I think we want something like the `${path.module}` magic [you mentioned before](https://github.com/chainguard-dev/terraform-provider-ko/issues/5#issuecomment-1178001994).

But this feels like an improvement anyway, so I can merge this and add a test case that uses modules somehow and see how it goes.